### DISCUSSION TITLE
2012/01/05/%E8%B1%86%E8%B1%86%E7%9A%842012%E5%85%83%E6%97%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#351] [Announcements]
# 2012/01/05/%E8%B1%86%E8%B1%86%E7%9A%842012%E5%85%83%E6%97%A6

豆豆这个元旦过得又非常开心，我们一家人回无锡过节去了。

https://blog.qizhen.xyz/2012/01/05/%E8%B1%86%E8%B1%86%E7%9A%842012%E5%85%83%E6%97%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

haiying
2012年01月6日 15:54 编辑
好爸爸!

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

7doudou
2012年01月10日 20:31 编辑
好巧，我也叫豆豆- -。
小豆豆，新年快乐~~~

回复
haiying
2012年01月7日 23:03 编辑
小豆豆越长越精神! 爸爸也很帅嘛

回复
ruanqizhen
2012年01月10日 10:45 编辑
豆豆毕竟从他妈那里继承了不少优秀基因，长得比我漂亮多了:)

回复
